Prototyper/Classes/Login/LoginView.swift

```diff
@@ -0,0 +1,128 @@
+//
+//  LoginView.swift
+//  KeychainSwift
+//
+//  Created by Raymond Pinto on 01.12.19.
+//
+
+import SwiftUI
+
+struct LoginView: View {
+    @EnvironmentObject var model: Model
+    @Environment(\.presentationMode) var presentationMode
+    @State var userid: String = ""
+    @State var password: String = ""
+    @State var showingAlert: Bool = false
+    @State var continueWithoutLoginState: Bool = false
+    @State var alertText: String = "Could not log in! Please check your login credentials and try again."
+    @State var userNamePlaceHolder = LoginViewConstants.userNamePlaceHolder.withoutLoginText
```

Comment:
You don't need to have `userNamePlaceHolder ` and `subButtonText` as an explicit `@State`s in the SwiftUI View as far as I can see. It should be fine if they are computed properties based on the `continueWithoutLogin` state that you already have:
```
var userNamePlaceHolder: String {
    if continueWithoutLogin {
          return LoginViewConstants.userNamePlaceHolder.withoutLoginText
    } else {
          return LoginViewConstants.userNamePlaceHolder.withLoginText
    }
}
```